pyproject.toml

```diff
@@ -1,20 +1,15 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "finch-tensor"
 version = "0.1.3"
```

Comment:
`setuptools_scm` would make this dynamic.

Reply:
Addressed in #2